src/Components/Result.jsx

```diff
@@ -1,23 +1,30 @@
-import React from "react";
+import { useRef } from "react";
 
-const Result = ({ userNum, computerNum }) => {
-  let result;
-  if (userNum) {
-    if (computerNum > userNum) {
-      result = "Your guess is too low!";
-    } else if (computerNum < userNum) {
-      result = "Your guess is too high";
-    } else if (computerNum == userNum) {
-      result = "Yay! You guessed correctly!";
-    } else {
-      result = "Enter valid input";
-    }
+const Result = ({ userNum }) => {
+  const randomNum = useRef(Math.floor(Math.random() * 100) + 1);
+  console.log(randomNum);
```

Comment:
I know I told you to add this console.log for debugging purposes, but unnecessary console.logs should be removed when you commit your code to git.